docs/notebooks/scverse_data_interoperability.ipynb

```diff
@@ -0,0 +1,139 @@
+{
```

Comment:
<p>Loom is actually implemented in AnnData (scanpy just re-exports that afaik), and there's also a <code>write_loom</code> function.  </p><p> </p><p>https://anndata.readthedocs.io/en/latest/api.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scverse-tutorials/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>